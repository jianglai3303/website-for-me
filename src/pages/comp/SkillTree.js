import React from "react"
import "../../styles/skill-tree.less"
export default class SkillTree extends React.Component {
    render(){
        return (
            <div className="skills-tree">
            <div className="skills-tree-wrapper">
                <h2 className="skills-tree-title">SKILLS<br/>LIST</h2>
                <div className="right-list">
                    <div className="skills-column">
                        <h3>BASICS</h3>
                        <ul>
                            <li>Linux, Git, Devops</li>
                            <li>AWS, Digital Ocean, Linode</li>
                        </ul>
                    </div>
                    <div className="skills-column">
                        <h3>Front End</h3>
                        <ul>
                            <li>HTML / JS / CSS / JQUERY</li>
                            <li>React / Vue / ANGULAR / Wordpress / Shopify</li>
                            <li>SEO / Responsive Website / High Efficiency</li>
                        </ul>
                    </div>

                    <div className="skills-column">
                        <h3>Back End</h3>
                        <ul>
                            <li>Node.js Nest.js Express Koa</li>
                            <li>MySQL, GraphQL, MangoDB</li>
                            <li>PYTHON PHP JAVA</li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
        )
    }
}