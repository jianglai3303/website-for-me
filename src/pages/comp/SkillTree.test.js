import React from "react"
import renderer from "react-test-renderer"

import SkillTree from "./SkillTree"

describe("SkillTree Section", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SkillTree />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
