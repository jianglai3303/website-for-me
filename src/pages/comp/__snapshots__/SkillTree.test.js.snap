// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkillTree Section renders correctly 1`] = `
<div>
  <p
    className="division"
  >
    WILLIAM LAI JIANG
  </p>
  <div
    className="skills-tree "
  >
    <div
      className="skills-tree-wrapper"
    >
      <h2
        className="skills-tree-title"
      >
        SKILLSÂ 
        <br />
        LIST
      </h2>
      <div
        className="right-list"
      >
        <div
          className="skills-column"
        >
          <h3>
            Front End
          </h3>
          <ul>
            <li>
              HTML, JS, CSS, Less, PostCss, Jquery
            </li>
            <li>
              React, MobX, Vue, Angular, RxJs, Wordpress, Shopify
            </li>
            <li>
              Npm , Webpack
            </li>
            <li>
              SVG, Three.js, CSS3, Barba.js, Sockit.io
            </li>
            <li>
              Mocha, Nightwatch
            </li>
            <li>
              SEO, Responsive Website, LightHouse
            </li>
          </ul>
        </div>
        <div
          className="skills-column"
        >
          <h3>
            Back End
          </h3>
          <ul>
            <li>
              Gatsby, Node.js, Nest.js, Django, Express, Koa
            </li>
            <li>
              GraphQL, NoSQL, MangoDB, MySQL
            </li>
            <li>
              CDN, Redis, Memcached
            </li>
            <li>
              Nginx, Apache, AWS, Digital Ocean, Linode
            </li>
            <li>
              Sentry, Uptimerobot
            </li>
          </ul>
        </div>
        <div
          className="skills-column"
        >
          <h3>
            BASICS
          </h3>
          <ul>
            <li>
              Docker, Linux, Git, Devops, CI, Jenkins
            </li>
            <li>
              Agile, Jira, Asana
            </li>
            <li>
              PhotoShop, Sketch
            </li>
          </ul>
        </div>
        <div
          className="skills-column"
        >
          <h3>
            Language
          </h3>
          <ul>
            <li>
              JavaScript, TypeScript, Python PHP 
            </li>
            <li>
              Java, C#, C++
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
